frontend/packages/ux-editor/src/components/config/editModal/EditCodeList.tsx

```diff
@@ -16,12 +31,21 @@ export function EditCodeList({ component, handleComponentChange }: IGenericEditC
 
   return (
     <div>
-      <TextField
-        id='modal-properties-code-list-id'
-        label={t('ux_editor.modal_properties_code_list_id')}
-        onChange={handleOptionsIdChange}
-        value={(component as FormCheckboxesComponent | FormRadioButtonsComponent).optionsId || ''}
-      />
+      {isLoadingCodeList ? (
+        <AltinnSpinner />
+      ) : isError ? (
+        <ErrorMessage>{t('ux_editor.modal_properties_error_message')}</ErrorMessage>
```

Comment:
Du trengte ikke å fjerne `error.message` her. Det er bra å kunne gi spesifikke feilmeldinger, det gjør det lettere for oss å feilsøke 🙂